docs-article-templates/src/helper/module-builder.ts

```diff
@@ -82,29 +84,46 @@ export function createModuleDirectory() {
 // data used to create the module yml file.
 // check settings.json for repo value.  if there's no value, the root path directory will be considered the repo name.
 export function updateModule(units) {
-    if (learnRepoId) {
-        learnRepo = learnRepoId;
-    } else {
+    if (!learnRepoId) {
         learnRepo = repoName;
     }
+    if (!gitHubID) {
+        author = `...`;
```

Comment:
Nit: `` ` `` isn't necessary

Reply:
But seems it's consistent so 👍 ? :)

Reply:
You're right, I'll add these to the cleanup task.